python/seldon_core/batch_processor.py

```diff
@@ -79,6 +81,15 @@ def start_multithreaded_batch_worker(
             "Batch size greater than 1 is only supported for `data` data type."
         )
 
+    # Providing call credentials sets the REST transport protocol to https,
+    # so we configure credentials even without a supplied token if use_ssl is set.
+    credentials = None
+    if use_ssl or len(call_credentials_token) > 0:
+        token = None
+        if len(call_credentials_token) > 0:
+            token = call_credentials_token
+        credentials = SeldonCallCredentials(token=token)
```

Comment:
It seems it may also require the SeldonChannelCredentials, as that would be required for GRPC